examples/cron/wrangler.toml

```diff
@@ -0,0 +1,13 @@
+name = "cron-worker"
+main = "build/worker/shim.mjs"
+compatibility_date = "2023-03-22"
+
+[build]
+command = "cargo install worker-build && worker-build --release"
+
+[triggers]
+crons = ["*/5 * * * *"]
```

Comment:
Would be nice to have a comment above to explain what the example cron does and link to the Cloudflare [cron docs](https://developers.cloudflare.com/workers/configuration/cron-triggers/).